_stbt/ocr.py

```diff
@@ -250,7 +250,7 @@ def ocr(frame=None, region=Region.ALL,
     if isinstance(tesseract_user_patterns, (bytes, str)):
         tesseract_user_patterns = [tesseract_user_patterns]
 
-    imglog = ImageLogger("ocr")
+    imglog = ImageLogger("ocr", result=None)
 
     text, region = _tesseract(
```

Comment:
Hmm, shouldn't `_tesseract` be returning `bytes`?.  IIRC the xml parser expects to receive bytes because XML can define the encoding inline.

Reply:
It works, I suppose because the xml library supports taking a str, and tesseract is always outputting UTF-8.